.github/workflows/schemacode_ci.yml

```diff
@@ -33,7 +33,7 @@ jobs:
       - name: "Install build dependencies"
         run: pip install --upgrade build twine
       - name: "Install test dependencies on tag"
-        run: pip install --upgrade pytest pyyaml pandas tabulate markdown-it-py
+        run: pip install --upgrade pytest lazy_loader
```

Comment:
Why install it here if it's a core dependency in `setup.cfg`?

Reply:
Because this doesn't install, it just builds. There's a weird build step where we use pytest to bundle the state of the BIDS examples into the repository so it gets pulled in by `python -m build`.

Reply:
I'm not sure I understand the build step, but thanks for clarifying.